orca-web/src/main/groovy/com/netflix/spinnaker/orca/Main.groovy

```diff
@@ -78,7 +79,8 @@ import org.springframework.scheduling.annotation.EnableAsync
   KayentaConfiguration,
   WebhookConfiguration,
   KeelConfiguration,
-  QosConfiguration
+  QosConfiguration,
+  GremlinConfiguration
```

Comment:
If you move `GremlinConfiguration` to `com.netflix.spinnaker.config`, it'll get picked up automatically and we don't need to touch anything outside of your `orca-gremlin` artifact.

This would be preferable.

Reply:
I changed the package name to `com.netflix.spinnaker.config`, but I still need the `orca-web` module to depend on `orca-integrations-gremlin`.  I also believe that I need to still have an explicit reference to `GremlinConfiguration` (parallel to `KeelConfiguration` / `QosConfiguration`)

I think I addressed all of the other comments.